src/ldc/attributes.d

```diff
@@ -223,3 +223,23 @@ immutable weak = _weak();
 private struct _weak
 {
 }
+
+///Readability aliases for compute
```

Comment:
how are these used?
If they are for use with `struct compute`, it's probably better to make them members of the struct.